src/core/Refiner.ml

```diff
@@ -795,6 +728,24 @@ struct
     in
     S.CodeExt (n, tfam, `Global tcof, tbdry)
 
+  let infer_nullary_ext : T.Chk.tac =
+    T.Chk.rule ~name:"Univ.infer_nullary_ext" @@ function
+      | ElStable (`Ext (0,code ,`Global (Cof (Meet [])),bdry)) -> 
```

Comment:
Same here, I think you need to allow this to match an arbitrary cofibration but then check whether it is true or not.